config/default.py

```diff
@@ -16,6 +16,7 @@
 SECURITY_PASSWORD_HASH = "bcrypt"
 SECURITY_PASSWORD_SALT = "abcde1234509876zyxwvu22"
 SECURITY_LOGIN_USER_TEMPLATE = "index.html"
+SECURITY_REGISTER_USER_TEMPLATE = "index.html"
```

Comment:
No, the modal doesn't pop up when you click "I don't have an account." It can't be done easily, and I think it's actually cleaner to see Login in the sidenav and Register in the view if you click on that.